promscale/send-data/jaeger.md

```diff
@@ -7,9 +7,37 @@ tags: [configure, traces]
 ---
 
 # Send Jaeger traces to Promscale
-Promscale natively supports the OpenTelemetry Protocol (OTLP) for traces.
-To ingest Jaeger traces to Promscale, use the OpenTelemetry Collector.
-The OpenTelemetry Collector converts Jaeger traces to OpenTelemetry traces.
+Promscale natively supports both Jaeger gRPC based remote storage(Promscale >= 0.14.0) and
+OpenTelemetry Protocol (OTLP) for traces.
```

Comment:
```suggestion
On Promscale&nbsp;0.14.0 and above, you can natively use both Jaeger 
gRPC based remote storage and OpenTelemetry Protocol (OTLP) for traces.
```

Reply:
Thanks @Loquacity for your suggestions.

"OpenTelemetry Protocol (OTLP) for traces" has been supported since 0.11.0. Hence the above suggested statement sounds incorrect.

Reply:
```suggestion
On Promscale&nbsp;0.14.0 and above, you can natively use Jaeger gRPC 
based remote storage. On Promscale&nbsp;0.11.0 and above, you can natively 
use OpenTelemetry Protocol (OTLP) for traces.
```

?